test-configs.yaml

```diff
@@ -685,6 +685,14 @@ device_types:
     filters:
       - blacklist: {defconfig: ['allnoconfig', 'allmodconfig']}
 
+# this is the same device than r8a7796-m3ulcb
+  r8a77960-ulcb:
```

Comment:
How about creating an alias for the existing device entry and reusing it here?

```yaml
  r8a7796-m3ulcb: &r8a7796-m3ulcb
    mach: renesas
[...]
  r8a77960-ulcb: *r8a7796-m3ulcb
```
and maybe do the same thing for the list of test plans if we want the same coverage on both?

Out of interest, what is the actual difference between `r8a7796-m3ulcb` and `r8a77960-ulcb` apart from the name?  Is it a new PCB version or that the dtb name changed in the kernel?

Reply:
The DTB name changed in the kernel.
For the alias let me check it